iceoryx_binding_c/test/moduletests/test_types_storage_size.hpp

```diff
@@ -15,68 +15,59 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+
+#ifndef IOX_TEST_TYPES_STORAGE_SIZE_HPP
+#define IOX_TEST_TYPES_STORAGE_SIZE_HPP
+
 #include "iceoryx_binding_c/internal/cpp2c_publisher.hpp"
 #include "iceoryx_binding_c/internal/cpp2c_subscriber.hpp"
 #include "iceoryx_posh/popo/listener.hpp"
 #include "iceoryx_posh/popo/untyped_client.hpp"
 #include "iceoryx_posh/popo/user_trigger.hpp"
 #include "iceoryx_posh/popo/wait_set.hpp"
 
-
-using namespace iox;
-using namespace iox::popo;
-
 extern "C" {
 #include "iceoryx_binding_c/types.h"
 }
 
 #include "test.hpp"
 
-namespace
+namespace iox_test_binding_c_types_storage_size
 {
 using namespace ::testing;
 
-TEST(iox_types_test, WaitSetStorageSizeFits)
+/* ######################################################
+ * Please add a test with this function in all test files
+ * with newly created storage types
+ * ######################################################*/
+
+inline void testBindingCTypesStorageSizes()
 {
```

Comment:
@elfenpiff @FerdinandSpitzschnueffler @dkroenke  since all of you are currently working on the C binding. I renamed `test_types.cpp` in `test_types_storage_sized.hpp` and created a function which needs to be added to a test for each new storage type. It's the second time this week that we are bitten by this and just get some weird segfaults. With this change, the storage size test will fail and give us an indicator whats going wrong